hl7-mllp/2.1/modules/ROOT/edits/hl7-mllp-connector-examples.adoc

```diff
@@ -0,0 +1,136 @@
+= HL7 EDI - Examples - Mule 4
+
+Use these examples to read and write HL7 messages over the MLLP protocol.
```

Comment:
```suggestion
Use these examples to read and write HL7 messages over Minimum Lower-Layer Protocol (MLLP).
```
Consider mentioning that there are prerequisites to using the examples and also telling what kind of example are included within.

Reply:
Also...does one "use" examples? "follow"? Is the audience simply reading and learning from what is written here, or implementing these?

Reply:
Fixed. It implies using them. I think this is pretty much standard for our example docs.